diff --git a/lib/manifest-path.js b/lib/manifest-path.js
index 972a0f4a5c970a26d13ebfc05db2184af42c3298..f5483eda2a07376fda5e02e4ed252f8e9ed671e4 100644
--- a/lib/manifest-path.js
+++ b/lib/manifest-path.js
@@ -11,7 +11,8 @@ export function viteManifestPath(
 	if (
 		vite.version.startsWith("5") ||
 		vite.version.startsWith("6") ||
-		vite.version.startsWith("7")
+		vite.version.startsWith("7") ||
+		vite.version.startsWith("8")
 	) {
 		return join(router.outDir, router.base, ".vite", "manifest.json");
 	}
